### PR TITLE
Send pdf email umbrella

### LINKED ISSUE #16: Add spinner (loading indicator) when user clicks "Zamów"

### LINKED ISSUE #19: List of orders - include order number on the list

### LINKED ISSUE #20: List of orders - show latest orders at the top

### PR DESCRIPTION
**Pull request by malczak**
☂️ PR
 closes #20
 closes #19
 closes #16 

+ adds 📄 PDF generator

### CODE REVIEW

File: components/Basket.tsx [hunk 1]
```diff
@@ -43,15 +44,20 @@ export default function Basket() {
...
   const onNextClick = () => {
     if (currentStep.onLeave) {
       currentStep.onLeave();
     }
     setStepIndex(stepIndex + 1);
   };
+
   const onOrder = () => {
     dispatch({ type: "CREATE_ORDER", brandId: config.id, client });
+    setIsBusy(true);
```
Comment (sobstel) [inline components/Basket.tsx:60]: Is it ever set back to false?

Comment (malczak) [reply components/Basket.tsx:60]: nope. thats a beauty of the matrix. by adding this single line we force universe to never ever fail on that action!

Comment (sobstel) [reply components/Basket.tsx:60]: 🤖 are you a real person?

Comment (malczak) [reply components/Basket.tsx:60]: +[--->++<]>+++.[->++++<]>+.[--->+<]>----.+[++>---<]>.++[--->++<]>.++++[->++<]>+.+[-->+<]>++.---[->+++<]>+.[->+++++<]>-.[->+++<]>+.-[->+++<]>.---[----->++<]>.-------------.----.+++++++++++.[++>---<]>--.-[--->++<]>--.[--->+<]>---.--------.------------.+++++++++++++.-[->+++++<]>-.[->+++<]>++.+++++++++++++.-----------.+[--->+<]>++.

Comment (sobstel) [reply components/Basket.tsx:60]: hum4n f135h. 0k. n07 5u23 480u7 8241n 7h0u9h.

**Pull request merged**